tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -147,18 +147,18 @@ Maybe Select Kernel
   Run Keyword If  not ${is_kernel_selected}  Click Button  xpath=//div[@class="jp-Dialog-buttonLabel"][.="Select"]/..
 
 Clean Up Server
-  Sleep  1
-  Maybe Close Popup
-  Navigate Home (Root folder) In JupyterLab Sidebar File Browser
-  Open With JupyterLab Menu  File  Close All Tabs
-  Maybe Close Popup
-  Sleep  1
-  Maybe Close Popup
-  Open With JupyterLab Menu  File  New  Notebook
-  Sleep  2
-  Maybe Close Popup
-  Add and Run JupyterLab Code Cell in Active Notebook  !rm -rf *
-  Wait Until JupyterLab Code Cell Is Not Active
+    [Documentation]    Cleans up user server and checks that everything has been removed
+    [Arguments]    ${admin_username}=${OCP_ADMIN_USER.USERNAME}    ${username}=${TEST_USER.USERNAME}
+    Maybe Close Popup
+    Run Keyword And Continue On Failure    Open With JupyterLab Menu    File    Close All Tabs
+    Maybe Close Popup
+    Clean Up User Notebook    ${admin_username}    ${username}
```

Comment:
I think before cleaning up it's better that you run `Navigate Home (Root folder) In JupyterLab Sidebar File Browser`  to avoid having a pop up message telling you that the current location is not found in the sidebar

Reply:
Done